### ISSUE TITLE
[Enhancement] Long swipe actions

### ISSUE THREAD

**Issue by jonasjensen-brolstark** [kirbydesign/designsystem#546]
**Describe the solution you'd like**
It should be possible to enable a "long swipe" (swiping the item over a certain threshold). Both left and right long swipe should be supported.

If there is more than 1 action it's always the 1st action (the action closest to the side of the list) that gets triggered on long swipe (e.g. "mark" in the example screenshot below).

AS of right now {N} does not support this. So this is blocked by [issue #1168](https://github.com/NativeScript/nativescript-ui-feedback/issues/1168) on the RadListView.

**Comment by labanos** [comment]
No longer blocked by {N}. Team 5 needs this after 1st release - so "parked" for now.

**Comment by jakobe** [comment]
This is supported out-of-the-box from Ionic, so should be fairly easy to implement:
https://ionicframework.com/docs/api/item-sliding#expandable-options

**Comment by alxzak** [comment]
This issue will be closed due to backlog prioritization and pruning.
https://github.com/kirbydesign/designsystem/discussions/1704#discussion-3522993
